bucket/ditto.json

```diff
@@ -1,18 +1,10 @@
 {
-    "version": "3.24.246.0",
+    "version": "3.25.76.0",
     "description": "An enhanced clipboard manager",
     "homepage": "https://ditto-cp.sourceforge.io",
     "license": "GPL-3.0-only",
-    "architecture": {
-        "64bit": {
-            "url": "https://github.com/sabrogden/Ditto/releases/download/3.24.246.0/DittoPortable_64bit_3_24_246_0.zip",
-            "hash": "d3e1e90d6cf8fa8fc067d718d150422f34feac40081137643b2987cd6da0f004"
-        },
-        "32bit": {
-            "url": "https://github.com/sabrogden/Ditto/releases/download/3.24.246.0/DittoPortable_3_24_246_0.zip",
-            "hash": "822c27f6cd73057b4ca2e71293fd53a6c308047cdd2aaef528c6aa7d3508624b"
-        }
-    },
+    "url": "https://github.com/sabrogden/Ditto/releases/download/3.25.76.0/DittoPortable_3_25_76_0.zip",
+    "hash": "sha256:86da50832e99c7063095d5a07f6b6b4bca8799e109f79bc64dbb6135d65f76d0",
```

Comment:
Just checked with the author, he said this is 64bit version so you need to wrap it inside architecture-64bit